### PR TITLE
Create ui-c8y-1019-20-0-extend-the-dynamic-forms-page-in-codex.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-3282](https://cumulocity.atlassian.net/browse/DM-3282)
Original PR: [6119](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6119)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1019-20-0-extend-the-dynamic-forms-page-in-codex.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Extend the "Dynamic forms" page in Codex
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-20-0-extend-the-dynamic-forms-page-in-codex.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Dynamic forms page in Codex extended with additional information
```

File: content/change-logs/application-enablement/ui-c8y-1019-20-0-extend-the-dynamic-forms-page-in-codex.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3282
+version: 1019.20.0
+---
+Extend the "Dynamic forms" page in Codex
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-20-0-extend-the-dynamic-forms-page-in-codex.md:17]: The Claude AI suggested to use the following body:
```suggestion
The Dynamic forms page in Codex lacked some important information to fully understand the feature. The page has been extended with additional details and examples to better explain the concept and usage of dynamic forms. This change makes it easier for users to comprehend how to utilize dynamic forms in their applications and provides a better reference for this feature in the documentation.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1019-20-0-extend-the-dynamic-forms-page-in-codex.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (bozhidardedov) [inline content/change-logs/application-enablement/ui-c8y-1019-20-0-extend-the-dynamic-forms-page-in-codex.md:17]:
```suggestion
The Dynamic forms page in Codex has been extended with additional details and examples to better explain the concept and usage of dynamic forms.
```

File: content/change-logs/application-enablement/ui-c8y-1019-20-0-extend-the-dynamic-forms-page-in-codex.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3282
+version: 1019.20.0
+---
+The Dynamic forms page in Codex has been extended with additional details and examples to better explain the concept and usage of dynamic forms.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1019-20-0-extend-the-dynamic-forms-page-in-codex.md:17]: @bozhidardedov Is this the exact name? Can´t find it, can you please provide the path?

Comment (bozhidardedov) [reply content/change-logs/application-enablement/ui-c8y-1019-20-0-extend-the-dynamic-forms-page-in-codex.md:17]: Yup, it's under Components > Forms > Dynamic forms.

Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1019-20-0-extend-the-dynamic-forms-page-in-codex.md:17]:
```suggestion
The Components > Forms > Dynamic forms page in the Codex has been extended with additional details and examples to better explain the concept and usage of dynamic forms.
```

**Review by BeateRixen — APPROVED** [review]

**Review by dafkk — APPROVED** [review]

**Pull request merged**